src/core/Microsoft.PowerToys.Settings.UI.Lib/ImageSize.cs

```diff
@@ -144,9 +144,17 @@ public double Width
 
             set
             {
-                if (_width != value)
+                int newWidth = -1;
+                int.TryParse(value + string.Empty, out newWidth);
+
+                if (newWidth < 0)
+                {
+                    newWidth = 0;
+                }
+
+                if (_width != value && newWidth >= 0)
```

Comment:
`newWidth >= 0` is a redundant check because there is an if statement above which sets `newWidth` to 0, if it is lesser than 0. Can we can remove it?

Reply:
thanks for the catch Alekhya!!!!

Reply:
I removed it here: https://github.com/microsoft/PowerToys/pull/4721/commits/e20dadb4e095a52fdc31f1993fcb82407daca6ff